web/crux/assets/install-script/install-docker.ps1.hbr

```diff
@@ -53,7 +53,7 @@ docker run `
   {{/if}}
   -e HOST_DOCKER_SOCK_PATH=//var/run/docker.sock `
   --add-host=host.docker.internal:host-gateway `
-  --name 'dagent' `
+  --name '{{#if workloadName}}{{workloadName}}{{else}}dagent{{/if}}' `
```

Comment:
would be nicer imho, if we could refrain to use handlebar conditions and move this if to the ts side. so the template simply can be workloadName